code/iaas/api-logic/src/main/java/io/cattle/platform/iaas/api/manager/VolumeManager.java

```diff
@@ -8,8 +8,16 @@
 import io.cattle.platform.util.type.CollectionUtils;
 import io.github.ibuildthecloud.gdapi.request.ApiRequest;
 
+import java.util.Map;
+
+import org.apache.commons.lang.StringUtils;
+
 public class VolumeManager extends AbstractJooqResourceManager {
 
+    private static final String NAME = "name";
+    private static final String URI = "uri";
+    private static final String DEVICE_NUM = "deviceNumber"; 
```

Comment:
Use io.cattle.platform.object.meta.ObjectMetaDataManager.NAME_FIELD

Move to VolumeConstants.FIELD_URI, etc...